CONTRIBUTING/RELEASING.md

```diff
@@ -330,6 +331,59 @@ Merging the pull request will trigger [the publish workflow](https://github.com/
 
 Done! 🚀
 
+## Releasing changes to older minor versions
+
+If you need to release a change to an older minor version that is not the latest, you have to do it manually, locally. The process is described below, with an example of releasing a new `v8.3.7` in a situation where `8.4.0` is currently the latest version.
+
+1. Checkout the _existing_ tag that matches the latest minor release you want to bump, and create a new branch from it. In this case, we want to do:
+  1. `git fetch --all --tags`
+  2. `git checkout tags/v8.3.6 -b patch-8-3-7`
+2. Make the changes you need to, most likely cherry-picking commits from the fix you need to back-port.
+3. Run `yarn install` in `scripts` and `code`
+4. Build all packages in `code` with `yarn task --task compile --no-link`
+5. Commit and push your changes.
+6. Trigger _daily_ CI manually on your branch:
+  1. Open https://app.circleci.com/pipelines/github/storybookjs/storybook
+  2. Click "Trigger Pipeline", top right corner
+  3. Pipeline: _"storybook default"_
+  4. Config Source: _"storybook"_
+  5. Branch: Your branch, eg. `patch-8-3-7`
+  6. Add a parameter, with _"name"_ `workflow`, _"value"_ `daily`
+7. Wait for CI to finish successfully.
+8. Bump all package versions:
+  1. `cd scripts`
+  2. `yarn release:version --release-type patch`
+9.  Commit with `git commit -m "Bump version from <CURRENT_VERSION> to <NEXT_VERSION> MANUALLY"`
+10. Add a new entry to `CHANGELOG.md`, describing your changes
+11. Commit with `git commit -m "Update CHANGELOG.md with <NEXT_VERSION> MANUALLY"`
+12. Ensure you have the correct write permissions to all the Storybook npm packages. You need to be an admin of the _storybook_ org, as well as the packages not in the org. The simplest way to check this, is to ensure you can see the _"Settings"_ tab in the follwing packages:
+  1.  [`@storybook/react-vite`](https://www.npmjs.com/package/@storybook/react-vite/access)
+  2.  [`storybook`](https://www.npmjs.com/package/storybook/access)
+  3.  [`sb`](https://www.npmjs.com/package/sb/access)
+  4.  [`create-storybook`](https://www.npmjs.com/package/create-storybook/access)
+13. Get your npm access token or generate a new one at https://www.npmjs.com/settings/jreinhold/tokens. Remember to not only give it access to the `storybook` org, but also the packages not in the org as listed above.
```

Comment:
@JReinhold, there is small item here for you to look into when you can (in this case, we should be agnostic and avoid including personal links to avoid issues).
```suggestion
13. Get your NPM access token or generate a new one at https://www.npmjs.com/settings/your-username/tokens. Remember to give it access to the `storybook` org and the packages not in the org, as listed above.
```